crates/consensus/src/consensus/mod.rs

```diff
@@ -111,7 +113,10 @@ impl HeaderStage {
             .or_panic()
     }
 
-    #[instrument(level = Level::INFO, skip(self))]
+    #[instrument(level = Level::DEBUG, skip_all,
```

Comment:
Why the switch back to DEBUG here? I thought that it's a significant enough (and occasional) event that it deserves being treated as log, no ?

Reply:
I thought this was a mistake as we/you changed all `instrument` to `DEBUG`.

Reply:
I really think we should not care about those levels in the code, but I will revert back to `INFO`

Reply:
Funny enough, I went for `DEBUG` in ADR-007 because i didn't see that `tracing` had a `TRACE` level. So I will be switching those `DEBUG` to `TRACE` in an upcoming commit. 

So that anything DEBUG/INFO will be treated as log. It'll be less confusing. 

Still for this one, I really think it shouldn't be a trace, but a log. A debug one perhaps but as you said, I don't think we should care about those levels for logging. Every log as INFO is good IMO.

Reply:
I am having a hard time making a difference between logs and traces, they are one and the same thing. the difference happens downstream with the use you make of that information: you can ignore timing and treat the entry or exit of a function as a log, being interested in the discrete event this represents, or you can care about the duration, or perhaps the fact it raised and error, and/or the call tree it produced, with or without timings. 

IMNSHO, those differences we make up-front are probably short-sighted and only breeds confusion and complexity in the code base. But I value more alignment within the team than opinion so I will make sure to read and follow the policy we agreed on in the ADR 👍

Reply:
We can summarize the difference simply:

- logs = for humans
- traces = for programs

As a human, I cannot process inputs on stdout when my terminal is flooded with thousands of lines per second. So of course, I can redirect the output, or export it to a sub-system for analysis/observability (which I believe is what you are advocating for).   

But, it's also nice to have some readily available output meant for immediate human-consumption. Something which I need not to spin up an telemetry backend with a collector and a trace visualizer to make sense of (which I believe, you have also been advocating for 😅 ...).